app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+
+  before_filter :require_logged_in
+
+  def create
+    @video = Video.find(params[:video_id])    
+    review = @video.reviews.new(set_params.merge!(user: current_user))
+    
+    if review.save
+      redirect_to @video
+    else
+      @reviews = @video.reviews.reload
+      render 'videos/show'
+    end
+  end
+
+  private
+
+  def set_params
```

Comment:
This method name is misleading. It sounds like you are changing the params. The general practice in naming strong param methods is `model_params`. In this case that would be `review_params`.